### PR TITLE
Added missing interface and create an executable upon build

### PR DESCRIPTION
**Pull request by amir734jj**

### CODE REVIEW

File: Deployr.Web.Contracts/ResponseContracts/DefaultResponse.cs [hunk 1]
```diff
@@ -4,11 +4,18 @@
 
 namespace Deployr.Web.Contracts.ResponseContracts
 {
+	public interface IDefaultResponse
```
Comment (TahaNKhan) [inline Deployr.Web.Contracts/ResponseContracts/DefaultResponse.cs]: So, I considered adding this but then realized it didn’t add any value. By the time people deserialize a contract, they need a concrete type, using an interface after that seems unnecessary.

Comment (amir734jj) [reply Deployr.Web.Contracts/ResponseContracts/DefaultResponse.cs]: This interface is needed otherwise code doesn't compile.

Comment (TahaNKhan) [reply Deployr.Web.Contracts/ResponseContracts/DefaultResponse.cs]: How so?

Comment (amir734jj) [reply Deployr.Web.Contracts/ResponseContracts/DefaultResponse.cs]: `dotnet build`, It will say the interface is missing ...

Comment (amir734jj) [reply Deployr.Web.Contracts/ResponseContracts/DefaultResponse.cs]: @TahaNKhan Removed the interface

**Review by TahaNKhan — APPROVED** [review]

**Pull request merged**